### DISCUSSION TITLE
What is the state of the master branch?

### DISCUSSION THREAD

**Discussion by klada** [heimdal/heimdal#1079] [General]
When looking at the latest releases I can see that even the recent 7.8 release from November '22 differs from the master branch by 2380 commits.

**When will the changes and fixes from the master branch be included in an official Heimdal release**? Since most issues are apparently only fixed in master it would be quite nice to have a release with things which have been fixed long time ago (such as #166, which has been fixed by fb81598d447305352cd38095ffac701cc3eed0cf over 4 years ago, but is still not included in any release).

**Comment by nicowilliams** [comment]
We're need:

 - fixes for `kinit` that unbreak some Windows-specific use cases
 - fixes for some principal aliasing backwards-incompatible changes in `lib/hdb`
 - more OpenSSL 3.0 ECDH/ECDSA/EDDSA fixes (for PKINIT)
 - misc fixes for important open issues

If I manage in time:

 - multi-master iprop

@jaltman, anything else missing from the above list?

**Comment by arekm** [comment]
Postgres currently (16) requires "MIT Credential Store" which exist in heimdal repository (https://github.com/heimdal/heimdal/issues/451) but with milestone set to heimdal 8. 

Do I assume correctly that heimdal 8 is not going to happen in foreseeable future, so mit krb 5 is the way to go in mean time for such things as postgresql?
